addons/binding/org.openhab.binding.enocean/ESH-INF/thing/channels.xml

```diff
@@ -154,6 +154,13 @@
         <description>Voltage of the battery</description>
         <state pattern="%f %unit%" readOnly="true" />
     </channel-type>
+	
+    <channel-type id="energyStorage">
+        <item-type>Number:ElectricPotential</item-type>
+        <label>Energy Storage</label>
+        <description>?</description>
+        <state pattern="%f %unit%" readOnly="true" />
+    </channel-type>
```

Comment:
Please use tabs for indentation in these XML files.

Reply:
Fixed, thanks!